swabbie-aws/src/test/java/com/netflix/spinnaker/swabbie/aws/images/AmazonImageHandlerTest.kt

```diff
@@ -364,6 +351,7 @@ object AmazonImageHandlerTest {
   @Test
   fun `should delete images`() {
     val fifteenDaysAgo = System.currentTimeMillis() - 15 * 24 * 60 * 60 * 1000L
+    val thirteenDaysAgo = System.currentTimeMillis() - 13 * 24 * 60 * 60 * 1000L
```

Comment:
Using real times in tests is asking for trouble. Should use a `Clock` with a fixed time, then use instants with calculated times for these sorta things.

Reply:
Arg! Missed that one. Fixing.